### PR TITLE
Gh 65

### LINKED ISSUE #65: Add module-tools

### PR DESCRIPTION
**Pull request by johnsonw**
Add module tools

Fixes #65.

Signed-off-by: Will Johnson <william.c.johnson@intel.com>

### CODE REVIEW

File: iml-online-help.spec [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+rm -rf %{buildroot}
+
+mkdir -p %{buildroot}/%{_prefix}/%{_lib}/iml-manager/%{name}
+cp -a dist/. %{buildroot}/%{_prefix}/%{_lib}/iml-manager/%{name}/
+
+%clean
+rm -rf %{buildroot}
+
+%files 
+%{buildroot}/%{_prefix}/%{_lib}/iml-manager/%{name}
```
Comment (jgrund) [inline iml-online-help.spec]: No buildroot here.

Comment (johnsonw) [reply iml-online-help.spec]: removed

File: iml-online-help.spec [hunk 2]
```diff
@@ -22,14 +22,14 @@ This module is a static html website based on the online-help markdown docs. The
...
-cp -a dist/. %{buildroot}/%{_prefix}/%{_lib}/iml-manager/%{name}/
+mkdir -p %{buildroot}%{_libdir}/iml-manager/%{name}
+cp -a dist/. %{buildroot}%{_libdir}/iml-manager/%{name}/
 
 %clean
 rm -rf %{buildroot}
 
 %files 
-%{buildroot}/%{_prefix}/%{_lib}/iml-manager/%{name}
+%dir %{buildroot}%{_libdir}/iml-manager/%{name}
```
Comment (jgrund) [inline iml-online-help.spec]: Still has `%{buildroot}/`

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,9 @@
+NAME                 := iml-online-help
+PACKAGE_VERSION      := 2.3.2
+DIST_VERSION         := $(PACKAGE_VERSION)
```
Comment (brianjmurrell) [inline Makefile]: This shouldn't be specified in the _Makefile_.  I know there are others that are doing it and they need to be fixed also.

Comment (johnsonw) [reply Makefile]: Brian, where should this live?

Comment (brianjmurrell) [reply Makefile]: Nowhere.  DIST_VERSION is used internally.

**Pull request merged**